addons/packages/packetfence.spec

```diff
@@ -488,7 +488,7 @@ done
 %{__install} -D -m0644 conf/systemd/packetfence-pfdhcp.service $RPM_BUILD_ROOT/usr/lib/systemd/system/packetfence-pfdhcp.service
 %{__install} -D -m0644 conf/systemd/packetfence-pfipset.service $RPM_BUILD_ROOT/usr/lib/systemd/system/packetfence-pfipset.service
 %{__install} -D -m0644 conf/systemd/packetfence-netdata.service $RPM_BUILD_ROOT/usr/lib/systemd/system/packetfence-netdata.service
-%{__install} -D -m0644 conf/systemd/packetfence-collectdtostatsd.service $RPM_BUILD_ROOT/usr/lib/systemd/system/packetfence-collectdtostatsd.service
+%{__install} -D -m0644 conf/systemd/packetfence-stats.service $RPM_BUILD_ROOT/usr/lib/systemd/system/packetfence-stats.service
```

Comment:
Its named pfstats in pf.conf but stats here

I believe those 2 need to be consistent